pom.xml

```diff
@@ -320,7 +321,7 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.75</minimum>
+                                            <minimum>0.70</minimum>
```

Comment:
Why do we decrease the min coverage ratio?

Reply:
added more tests, min is set to 0.75 now